src/vs/workbench/services/localization/browser/languagePackTranslationsService.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Language } from 'vs/base/common/platform';
+import { URI } from 'vs/base/common/uri';
+import { getGalleryExtensionId } from 'vs/platform/extensionManagement/common/extensionManagementUtil';
+import { ITranslations, localizeManifest } from 'vs/platform/extensionManagement/common/extensionNls';
+import { IExtensionManifest } from 'vs/platform/extensions/common/extensions';
+import { registerSingleton } from 'vs/platform/instantiation/common/extensions';
+import { IProductService } from 'vs/platform/product/common/productService';
+import { IExtensionResourceLoaderService } from 'vs/workbench/services/extensionResourceLoader/common/extensionResourceLoader';
+import { ILanguagePackTranslationsService } from 'vs/workbench/services/localization/common/languagePackTranslations';
+
+export class LanguagePackTranslationsService implements ILanguagePackTranslationsService {
```

Comment:
Just curious to know why this is a service? I see this is only used in builtin extensions and seems very tightly associated to builtin extensions. Is there a scenario where this can be used at other places too? If not, shall we inline this in builtin extensions scanner service?